### ISSUE TITLE
문헌링크 작업 건

### ISSUE THREAD

**Issue by mkh2009** [no05s/pdf-support#24]
원문
![20190620_225419](https://user-images.githubusercontent.com/51990901/59854932-d80f8880-93ae-11e9-92a2-57443d42e55a.png)
 
참고문헌
![20190620_225458](https://user-images.githubusercontent.com/51990901/59854966-f07fa300-93ae-11e9-88c4-acd92f9522ad.png)

법원도서관
![KakaoTalk_20190620_224403961](https://user-images.githubusercontent.com/51990901/59855016-0b521780-93af-11e9-8e46-4d09d5c9903b.png)

원문의 문헌과 첨고문헌의 년도가 다를경우 참고문헌의 링크는 내용에 적힌대로 79년도
작업해서 링크를 걸어야 하나요?   원문문헌은 92년도로 링크작업 했습니다.

**Comment by dhkim11** [comment]
네, 원문에 언급된 내용 그대로 링크작업하시면 됩니다. 참고문헌에서 1979년도로 언급되었으면, 1979년도의 문헌으로 링크작업해주시면 됩니다.
